File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
import sys
input = sys.stdin.readline

arr = {}
total = 0
for _ in range(3):
    x = int(input())
    if x in arr:
        arr[x] += 1
        total += x
    else:
        arr[x] = 1
        total += x

if total == 180:
    if len(arr) == 1:
        print("Equilateral")
    elif len(arr) == 2:
        print("Isosceles")
    else:
        print("Scalene")
else:
    print("Error")

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.js
let fs = require('fs')
let input = fs.readFileSync("/dev/stdin").toString().trim().split("\n")
// let input = fs.readFileSync("./EX.txt").toString().trim().split("\n")

let arr = []
let total = 0
input.forEach(x => {
    const num = parseInt(x)
    if (arr.includes(num) === false) {
        arr.push(num)
        total += num
    } else {
        total += num
    }
})

if (total === 180) {
    if (arr.length === 1) {
        console.log("Equilateral")
    } else if (arr.length === 2) {
        console.log("Isosceles")
    } else {
        console.log("Scalene")
    }
} else {
    console.log("Error")
}